sample/OSCON17/sessions

```diff
@@ -0,0 +1,548 @@
+[{
+		"id": 1,
+		"title": "Wednesday opening welcome",
+		"subtitle": null,
+		"short_abstract": null,
+		"long_abstract": "Program chairs Rachel Roumeliotis and Kelsey Hightower open the first day of keynotes.",
+		"start_time": "2017-05-10T09:00:00",
+		"end_time": "2017-05-10T09:05:00",
+		"session_type": {
+
+		},
+		"track": {
```

Comment:
This does cause a crash. It saves this track object in the database -

track = new Track();
track.name = null
track.color = null
track.id = 0

And this causes the reactive db notification to hit up to show this track on the track fragment but as everything is null, app crashes

Don't use empty objects to denote no track, use null